pkg/trivy/wrapper_mock.go

```diff
@@ -1,13 +1,18 @@
 package trivy
 
 import (
+	"github.com/aquasecurity/trivy/pkg/types"
 	"github.com/stretchr/testify/mock"
 )
 
 type MockWrapper struct {
 	mock.Mock
 }
 
+func (w *MockWrapper) GetVersion() (types.VersionInfo, error) {
+	panic("implement me!")
```

Comment:
Don't panic! :) Use mocked GetVersion method in HTTP handler test(s)

Reply:
Fixed https://github.com/aquasecurity/harbor-scanner-trivy/pull/87/commits/3740f424a1a70b1fd880c35e197f32d721846b68